integration-tests-ex/tools/src/main/java/org/apache/druid/testing/tools/AvroSchemaRegistryEventSerializer.java

```diff
@@ -49,6 +48,22 @@ public class AvroSchemaRegistryEventSerializer extends AvroEventSerializer
 
   private Schema fromRegistry;
 
+  public AvroSchemaRegistryEventSerializer(
+      String schemaRegistryHost
+  )
+  {
+    this.config = null;
+    this.client = new CachedSchemaRegistryClient(
+        StringUtils.format("http://%s", schemaRegistryHost),
+        Integer.MAX_VALUE,
+        ImmutableMap.of(
+            "basic.auth.credentials.source", "USER_INFO",
+            "basic.auth.user.info", "druid:diurd"
+        ),
+        ImmutableMap.of()
+    );
```

Comment:
This snippet can probably be put in a utility method, in say `KafkaUtil`, as it seems to be used in multiple places.